### PR TITLE
UI: Remove LearnLink, use DocLink!

### PR DESCRIPTION
**Pull request by hellobontempo**

### CODE REVIEW

File: ui/app/templates/components/license-banners.hbs [hunk 1]
```diff
@@ -31,9 +31,9 @@
       }}"
       @marginless={{true}}
     >
-      <LearnLink @path="/tutorials/nomad/hashicorp-enterprise-license">
```
Comment (hellobontempo) [inline ui/app/templates/components/license-banners.hbs:34]: Not sure why we were referencing Nomad docs here, was this intentional?

Comment (hashishaw) [reply ui/app/templates/components/license-banners.hbs:34]: I think vault didn't have a dedicated license page yet, so we used nomad's which has the same instructions for generating a license

File: ui/app/templates/components/wizard/replication-setup.hbs [hunk 2]
```diff
@@ -11,15 +11,15 @@
       more interested in speed of access?
     </p>
 
-    <LearnLink @path="/vault/operations/ops-replication">
+    <DocLink @path="/vault/operations/ops-replication">
```
Comment (hashishaw) [inline ui/app/templates/components/wizard/replication-setup.hbs:14]: [This page](https://developer.hashicorp.com/vault/operations/ops-replication) doesn't seem to exist

**Review by hashishaw — APPROVED** [review]
Great work getting through all of these! I just found one that needs to be updated, but after that it's good to go 🙏

File: ui/app/templates/components/wizard/replication-setup.hbs (hunk 2 above)
Comment (hellobontempo) [reply ui/app/templates/components/wizard/replication-setup.hbs:14]: Good catch! Thank you - the path should be `/vault/tutorials/enterprise/performance-replication`

Comment (hellobontempo) [inline ui/app/templates/components/wizard/replication-setup.hbs:14]:
```suggestion
    <DocLink @path="/vault/tutorials/enterprise/performance-replication">
```

**Pull request merged**